website/content/en/docs/faq.md

```diff
@@ -211,7 +211,7 @@ For information on upgrading Karpenter, see the [Upgrade Guide]({{< ref "./upgra
 
 ### How do I upgrade an EKS Cluster with Karpenter?
 
-When upgrading an Amazon EKS cluster, [Karpenter's Drift feature]({{<ref "./concepts/disruption#drift" >}}) can automatically upgrade the Karpenter-provisioned nodes to stay in-sync with the EKS control plane. Karpenter Drift currently needs to be enabled using a [feature gate]({{<ref "./reference/settings#feature-gates" >}}).
+When upgrading an Amazon EKS cluster, [Karpenter's Drift feature]({{<ref "./concepts/disruption#drift" >}}) can automatically upgrade the Karpenter-provisioned nodes to stay in-sync with the EKS control plane. Karpenter Drift is enabled by default starting v0.33.x using `Drift=true` in [feature gate]({{<ref "./reference/settings#feature-gates" >}}).
```

Comment:
nit: since Drift is now enabled by default I'm not sure there's a need to call out the feature flag here at all. 
```suggestion
When upgrading an Amazon EKS cluster, [Karpenter's Drift feature]({{<ref "./concepts/disruption#drift" >}}) can automatically upgrade the Karpenter-provisioned nodes to stay in-sync with the EKS control plane. 
```

Reply:
Makes sense. I have got rid of the feature gate mention to keep it simple.

Reply:
I agree with the wording, "Karpenter drift is enabled by default starting in v0.33.x."; however, can we still link out to the feature gate reference docs? This really feels like it needs a link to point where we document that it's on by default and then to also show users how they can disable it if they want to

Reply:
Discussed offline: This change seems find to me.